Fix to put docs in correct location based on ${PF}
This requires -DPF=${PF} passed to cmake

Also prevent the installation of COPYING

Patch by Kevin McCarthy <signals42@gmail.com>


--- CMakeLists.txt
+++ CMakeLists.txt
@@ -83,12 +83,12 @@
 	add_custom_target(pdf DEPENDS teapot.pdf)
 	add_custom_target(html DEPENDS html/index.html)
 	add_custom_target(doc ALL DEPENDS teapot.pdf html/index.html)
-	install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/html DESTINATION share/doc/teapot FILES_MATCHING PATTERN *.html PATTERN *.png)
-	install(FILES ${CMAKE_CURRENT_BINARY_DIR}/teapot.pdf DESTINATION share/doc/teapot)
-	set(HELPFILE "${CMAKE_INSTALL_PREFIX}/share/doc/teapot/html/index.html")
+	install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/html DESTINATION share/doc/${PF} FILES_MATCHING PATTERN *.html PATTERN *.png)
+	install(FILES ${CMAKE_CURRENT_BINARY_DIR}/teapot.pdf DESTINATION share/doc/${PF})
+	set(HELPFILE "/usr/share/doc/${PF}/html/index.html")
 endif ()
 
-install(FILES COPYING README DESTINATION share/doc/teapot)
+install(FILES README DESTINATION share/doc/${PF})
 
 add_custom_target(dist
 	COMMAND ${CMAKE_COMMAND} -E remove_directory teapot-${Teapot_VERSION_MAJOR}.${Teapot_VERSION_MINOR}.${Teapot_VERSION_PATCH}
